app/metrics.py

```
@@ -0,0 +1,20 @@
+"""Prometheus metrics for Barua Pepe"""
+
+try:
+    from prometheus_client import Counter, generate_latest, CONTENT_TYPE_LATEST
+except Exception:  # pragma: no cover - optional in environments without prometheus_client
+    generate_latest = None
+    CONTENT_TYPE_LATEST = "text/plain; version=0.0.4"
+
+    class Counter:
```

Pylint (4.0.6)

[refactor] Too few public methods (1/2)

(R0903)